src/BackupServiceProvider.php

```diff
@@ -45,5 +48,16 @@ public function packageRegistered()
         $this->app->singleton(ConsoleOutput::class);
 
         $this->app->bind(CleanupStrategy::class, config('backup.cleanup.strategy'));
+
+        $this->registerDiscordChannel();
+    }
+
+    protected function registerDiscordChannel()
+    {
+        Notification::resolved(function (ChannelManager $service) {
+            $service->extend('discord', function ($app) {
```

Comment:
Could this conflict with the discord notification channel? 🤔
https://github.com/laravel-notification-channels/discord

Reply:
Mentioned package does not seem be registering something like this inside the ServiceProvider.
[https://github.com/laravel-notification-channels/discord/blob/master/src/DiscordServiceProvider.php](https://github.com/laravel-notification-channels/discord/blob/master/src/DiscordServiceProvider.php)